src/updatelikes.js

```diff
@@ -0,0 +1,10 @@
+import { getMovielike } from './InvolvementAPI.js';
+import { grab } from './utills.js';
+
+export default async function showlikes(movieId) {
+  const resp = await getMovielike(movieId);
+  const likesArray = JSON.parse(resp);
+  const likedMovie = likesArray.find((obj) => obj.item_id === movieId);
+  const DomLike = grab(movieId);
+  DomLike.innerText = `${likedMovie.likes} Like ( s )`;
+}
```

Comment:
- [ ] Use ES6 syntax (`arrow function`) instead of using `function` keyword. arrow functions, introduced in `ES6`, and provides a concise way to write functions in JavaScript.